packages/eslint-config-react-native/index.js

```diff
@@ -1,119 +1,20 @@
 /**
  * strvcom/eslint-config-react-native
  *
- * @author      André Nanninga <andre.nanninga@strv.com>
- * @copyright   2019 STRV
+ * @author      Petr Konecny <petr.konecny@strv.com>
+ * @copyright   2016-present Expo, 2023-present STRV
```

Comment:
Since this file was 100% authored by people contractually bound or directly employed by STRV and does not contain verbatim copies of any 3rd party code I think the copyright should belong to STRV in full.